README.md

```diff
@@ -1,56 +1,24 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# To Do list
```

Comment:
- [ ] Let's try to be as detailed as possible in our documentation as it can be of great benefit to users. In your README, kindly add instructions on how to set up and run the app locally ie from cloning the app to running `npm install` to running `npm start` (or `npm run build`, then open `dist/index.html` file).

- [ ] Also add the **Prerequisites** section and list the technologies users need to have to run the app.

Reply:
Thanks for the tip.